recipes/octo-wildcardmatching-cpp/all/conanfile.py

```diff
@@ -0,0 +1,89 @@
+
+    def validate(self):
+        if self.info.settings.compiler.cppstd:
+            check_min_cppstd(self, "17")
+
+        minimum_version = self._compilers_minimum_version.get(str(self.info.settings.compiler), False)
+        if minimum_version and Version(self.info.settings.compiler.version) < minimum_version:
+            raise ConanInvalidConfiguration(
+                f"{self.name} requires C++17, which your compiler does not support."
+            )
+        else:
+            self.output.warn(f"{self.name} requires C++17. Your compiler is unknown. Assuming it supports C++17.")
+        if self.settings.compiler == "clang" and self.settings.compiler.get_safe("libcxx") == "libc++":
+            raise ConanInvalidConfiguration(f"{self.name} does not support clang with libc++. Use libstdc++ instead.")
+        if self.settings.compiler == "Visual Studio" and self.settings.compiler.runtime in ["MTd", "MT"]:
+            raise ConanInvalidConfiguration(f"{self.name} does not support MSVC MT/MTd configurations, only MD/MDd is supported")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version],
+            destination=self.source_folder, strip_root=True)
+
+    def build_requirements(self):
+        self.build_requires("cmake/3.24.0")
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def package(self):
+        copy(self, "LICENSE", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
+        cmake = CMake(self)
+        cmake.install()
+
+    def package_info(self):
+        self.cpp_info.set_property("cmake_file_name", "octo-wildcardmatching-cpp")
+        self.cpp_info.set_property("cmake_target_name", "octo::octo-wildcardmatching-cpp")
+        self.cpp_info.set_property("pkg_config_name", "octo-wildcardmatching-cpp")
+        self.cpp_info.components["libocto-wildcardmatching-cpp"].libs = ["octo-wildcardmatching-cpp"]
+        self.cpp_info.components["libocto-wildcardmatching-cpp"].requires = []
+        self.cpp_info.filenames["cmake_find_package"] = "octo-wildcardmatching-cpp"
+        self.cpp_info.filenames["cmake_find_package_multi"] = "octo-wildcardmatching-cpp"
+        self.cpp_info.names["cmake_find_package"] = "octo-wildcardmatching-cpp"
+        self.cpp_info.names["cmake_find_package_multi"] = "octo-wildcardmatching-cpp"
+        self.cpp_info.names["pkg_config"] = "octo-wildcardmatching-cpp"
+        self.cpp_info.components["libocto-wildcardmatching-cpp"].names["cmake_find_package"] = "octo-wildcardmatching-cpp"
+        self.cpp_info.components["libocto-wildcardmatching-cpp"].names["cmake_find_package_multi"] = "octo-wildcardmatching-cpp"
+        self.cpp_info.components["libocto-wildcardmatching-cpp"].set_property("cmake_target_name", "octo::octo-wildcardmatching-cpp")
+        self.cpp_info.components["libocto-wildcardmatching-cpp"].set_property("pkg_config_name", "octo-wildcardmatching-cpp")
```

Comment:
These are way complicated, v1 and v2 targets dont match FYI

I noticed a few of your other PRs all use the same template but it would be nice to fix them

```suggestion
    def package_info(self):
        self.cpp_info.set_property("cmake_file_name", "octo-wildcardmatching-cpp")
        self.cpp_info.set_property("cmake_target_name", "octo::octo-wildcardmatching-cpp")
        self.cpp_info.set_property("pkg_config_name", "octo-wildcardmatching-cpp")
        self.cpp_info.libs = ["octo-wildcardmatching-cpp"]
        self.cpp_info.names["cmake_find_package"] = "octo-wildcardmatching-cpp"
        self.cpp_info.names["cmake_find_package_multi"] = "octo-wildcardmatching-cpp"
        self.cpp_info.names["pkg_config"] = "octo-wildcardmatching-cpp"
```

Reply:
It makes sense!

Reply:
https://github.com/conan-io/conan-center-index/pull/12466
https://github.com/conan-io/conan-center-index/pull/12465

Reply:
You are an amazing contributor! 

Amazing work ❤️